mkdocs.yml

```diff
@@ -106,21 +106,10 @@ nav:
       - Notifications: "customize/notifications.md"
   - Develop:
       - Overview: "develop/index.md"
-      - Getting started:
-          - Source code: "develop/getting-started/source-code.md"
-          - Package development: "develop/getting-started/package-development.md"
-          - Instance development: "develop/getting-started/instance-development.md"
-          - Debugging: "develop/getting-started/debugging.md"
-          - Coding style: "develop/getting-started/code-style.md"
-          - Virtual environments: "develop/getting-started/virtualenvs.md"
-          - Getting help: "develop/getting-started/help.md"
-          - Development process: develop/process.md
-      - Best practices:
-          - Accessibility (a11y): develop/best-practices/accessibility.md
-          - Commits, PRs & reviews: develop/best-practices/commits.md
-          - CSS/JavaScript: develop/best-practices/css-js.md
-          - React: develop/best-practices/react.md
-          - User interface: develop/best-practices/ui.md
+      - Getting Started:
```

Comment:
This section will eventually become the "Operate" section for how people running an instance can configure / customize / enhance it.  Inevitably will require them to deal with virtual envs, debug and seek help ...